packages/tools/src/tools/displayTools/Contour/contourDisplay.ts

```diff
@@ -95,89 +91,19 @@ async function render(
     );
   }
 
-  // From here to below it is basically the legacy geometryId based
-  // contour rendering via vtkActors that has some bugs for display,
-  // as it sometimes appear and sometimes not, and it is not clear.
-  // We have moved to the new SVG based contours via our annotation tools
-  // check out annotationUIDsMap in the ContourSegmentationData type
-  const { geometryIds } = contourData;
-
-  if (!geometryIds?.length || !(viewport instanceof BaseVolumeViewport)) {
+  if (!contourData) {
     return;
   }
 
-  // add the contour sets to the viewport
-  addOrUpdateVTKContourSets(
-    viewport,
-    geometryIds,
-    representationConfig,
-    toolGroupConfig
-  );
-
-  /**
-   * The following logic could be added if we want to support the use case
-   * where the contour representation data is initiated using annotations
-   * in the state from the get-go , and not when the user draws a contour.
-   */
-  // if (contourData?.points?.length) {
-  //   // contourData = createAnnotationsFromPoints(contourData.points);
-  //   const contourSegmentationAnnotation = {
-  //     annotationUID: csUtils.uuidv4(),
-  //     data: {
-  //       contour: {
-  //         closed: true,
-  //         polyline: contourData.points,
-  //       },
-  //       segmentation: {
-  //         segmentationId,
-  //         segmentIndex: 1, // Todo
-  //         segmentationRepresentationUID:
-  //           representationConfig.segmentationRepresentationUID,
-  //       },
-  //     },
-  //     highlighted: false,
-  //     invalidated: false,
-  //     isLocked: false,
-  //     isVisible: true,
-  //     metadata: {
-  //       toolName: 'PlanarFreehandContourSegmentationTool',
-  //       FrameOfReferenceUID: viewport.getFrameOfReferenceUID(),
-  //       viewPlaneNormal: viewport.getCamera().viewPlaneNormal,
-  //     },
-  //   };
-
-  //   addAnnotation(contourSegmentationAnnotation, viewport.element);
-  // } else if (
-  //   !contourData &&
-  //   polySeg.canComputeRequestedRepresentation(
-  //     representationConfig.segmentationRepresentationUID
-  //   )
-  // ) {
-  // contourData = await polySeg.computeAndAddContourRepresentation(
-  //   segmentationId,
-  //   {
-  //     segmentationRepresentationUID:
-  //       representationConfig.segmentationRepresentationUID,
-  //     viewport,
-  //   }
-  // );
-  // }
-
-  // if (contourData?.geometryIds?.length) {
-  //   handleVTKContour({
-  //     viewport,
-  //     representationConfig,
-  //     toolGroupConfig,
-  //     geometryIds: contourData.geometryIds,
-  //   });
-  // } else if (contourData.annotationUIDsMap?.size) {
-  //   handleContourAnnotationSegmentation({
-  //     viewport,
-  //     representationConfig,
-  //     toolGroupConfig,
-  //     annotationUIDsMap: contourData.annotationUIDsMap,
-  //   });
-  // }
+  if (contourData?.geometryIds?.length) {
```

Comment:
let's keep the vtk path too, can we ?

Reply:
there's only 1 path for both

Reply:
review my messages and the PR description for this, we can do something else if its not ideal, I think its better to keep it like this